### PR TITLE
upgrade frost-core to 1.0.0

### PR DESCRIPTION
**Pull request by rox163**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [x] #major# - incompatible API change

# CHANGELOG
**upgraded** frost-core to 1.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,11 +45,11 @@
     "ember-cli-uglify": "^1.2.0",
```
Comment (dafortin) [inline package.json]: Can you remove app version

File: package.json [hunk 2]
```diff
@@ -37,34 +37,35 @@
...
     "ember-disable-proxy-controllers": "^1.0.1",
+    "ember-elsewhere": "~0.4.1",
     "ember-export-application-global": "^1.0.4",
-    "ember-frost-core": "~0.29.0",
-    "ember-hook": "~1.3.5",
+    "ember-frost-core": "^1.0.0",
+    "ember-hook": "^1.3.5",
     "ember-load-initializers": "^0.5.0",
     "ember-lodash": "~0.0.7",
     "ember-one-way-controls": "0.8.2",
```
Comment (dafortin) [inline package.json]: Can you remove this

File: bower.json [hunk 3]
```diff
@@ -5,13 +5,15 @@
...
     "ember-inflector": "1.3.1",
     "ember-mocha": "~0.8.11",
+    "ember-mocha-adapter": "~0.3.1",
+    "es6-promise": "^3.2.2",
     "highlightjs": "^9.2.0",
     "lodash": "^4.10.0",
-    "perfect-scrollbar": ">=0.6.7 <2.0.0",
-    "sinonjs": "1.17.1",
     "marked": "^0.3.5",
+    "perfect-scrollbar": ">=0.6.7 <2.0.0",
```
Comment (dafortin) [inline bower.json]: Do we still need this?

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
## Changed

### Chrome: Basic-File-Picker
<table><tr> <td></td> <td></td> <td></td> </tr><tr> <td>Diff</td> <td>Current</td> <td>Baseline</td> </tr></table>
### Chrome: File-upload
<table><tr> <td></td> <td></td> <td></td> </tr><tr> <td>Diff</td> <td>Current</td> <td>Baseline</td> </tr></table>

File: bower.json (hunk 3 above)
Comment (rox163) [inline bower.json]: what is replacing the perfect scroll?

Comment (dafortin) [inline bower.json]: This was imported by core?

**Pull request merged**